src/schema/objects/suffixes.yaml

```diff
@@ -669,6 +669,17 @@ meg:
   description: |
     Unprocessed MEG data stored in the native file format of the MEG instrument
     with which the data was collected.
+models:
```

Comment:
Should this be `model` in singular?

Reply:
I don't think so, the file is called `_models.tsv`.

As opposed to that, there will be a new `model` entry in this file in the context of #1282.